### DISCUSSION TITLE
Relative expression profiles

### DISCUSSION THREAD

**Discussion by RickHH** [Open-Systems-Pharmacology/Forum#1629] [Questions & Problems] [answered]
Hi all,

How can I modify the relative expression ratio of partner proteins in different organs in MoBi?
I set the relative expression of the partner protein in PK-sim, but when I import the simulation into MoBi, I can only find the following parameters under molecule module and don't know how to adjust its relative expression in different organs. Although I saw a "relative expression" parameter, I can't figure out how to set it for different organs.

Thanks
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/39480454/5628870a-d1e2-4930-95cb-d41fc2f6ad86)

**Comment by PavelBal** [comment, marked as answer]
These parameters are *local*, that means, the parameter has different values in different organs/compartments. You will find these parameters in the **Parameters Start Values**  building block:

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25903040/e80c4e22-8f2e-4375-a0f9-312636d09130)

**Reply by RickHH** [reply]
Hi Pavel, thank you so much for the reply, I got it.
